src/test/cloudWatchLogs/commands/searchLogGroup.test.ts

```diff
@@ -45,4 +45,23 @@ describe('searchLogGroup', async function () {
         accept(testInput)
         assert.strictEqual(await result, testInput)
     })
+
+    it('wizaard prioritizes logGroupInfo if passed in', async function () {
```

Comment:
```suggestion
    it('wizard prioritizes logGroupInfo if passed in', async function () {
```
May want to consider a different title. What does this mean in terms of what the user sees?